File: LV1/1.4/parking2/parking2.cpp
#include <cstdio>

int main() {
    int n, m, tmp;
    scanf("%d", &n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &m);

        int max = 0, min = 1000;
        for (int j = 0; j < m; j++) {
            scanf("%d", &tmp);
            if (tmp > max) {
                max = tmp;
            }
            if (tmp < min) {
                min = tmp;
            }
        }
        printf("%d\n", 2 * (max - min));
    }
}

File: LV1/1.4/parking2/parking2.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();
    let mut buff = stdin.lock().lines().map(|l| l.unwrap());
    let n: i32 = buff.next().unwrap().trim().parse().unwrap();

    for _ in 0..n {
        buff.next();
        let park: Vec<i32> = buff
            .next()
            .unwrap()
            .split_whitespace()
            .map(|n| n.parse().unwrap())
            .collect();

        let max = park.iter().max().unwrap();
        let min = park.iter().min().unwrap();

        println!("{}", 2 * (max - min));
    }
}
